profiles/apparmor.d/usr.share.openqa.script.worker

```diff
@@ -119,6 +119,8 @@
 
 
   profile /usr/bin/Xvnc {
+    /home/newman/openqa/.../os-autoinst/bmwqemu.pm r,
```

Comment:
This looks odd...

Reply:
indeed, possibly also https://github.com/os-autoinst/openQA/pull/1599/files#diff-d9bf49a0805be292590c49c2cc771821R214